package.json

```diff
@@ -64,7 +64,7 @@
   "devDependencies": {
     "@commitlint/cli": "^19.3.0",
     "@commitlint/config-conventional": "^19.2.2",
-    "@nestjs-modules/mailer": "^2.0.2",
+    "@nestjs-modules/mailer": "^1.6.1",
```

Comment:
Why down grading the package?

Reply:
I. Will install packages using npm install

Reply:
I was installing packages using npm install but I get errors of deprecated packages